Mathlib/AlgebraicTopology/SimplexCategory.lean

```diff
@@ -30,6 +31,13 @@ We provide the following functions to work with these objects:
 4. `SimplexCategory.Hom.toOrderHom` gives the underlying monotone map associated to a
   term of `SimplexCategory.Hom`.
 
+## Notations
+
+* `[n]` denotes the `n`-dimensional simplex. This notation is available with
+  `open Simplicial`.
+* `[m]ₙ` denotes the `m`-dimensional simplex in the `n`-truncated simplex category.
+  The truncation proof `p : m ≤ n` can also be provided using the syntax `[m, p]ₙ`.
+  This notation is available with `open SimplexCategory.Truncated`.
```

Comment:
I remain opposed to introducing any notations that conflict with `List` notation. This is not on any technical grounds, but simply that they *reduce readability* relative to no notation.

Reply:
@kim-em to be sure I understand, could you please clarify which of the following you are suggesting?
- Remove the existing notation `[n]` completely.
- Remove the `(priority := high)` added to the `[n]` notation declaration in this PR.
- Change the proposed notation `[m]ₙ`, added in this PR.
- Get rid of both notations `[n]` and `[m]ₙ`.

Reply:
I would suggest changing the existing notation `[_]` (as it is barely usable because of the conflict with the list notation) and change `[_]ₙ` accordingly.

Reply:
It could be a good idea to seek advice/suggestions about alternative notations on Zulip?

Reply:
Agreed. Thanks all for the suggestions. I will get a Zulip thread going and see if others are willing to weigh in.

Reply:
The notation for objects of the simplex category has been changed in #21565. I will update this PR to reflect the same change for objects of the truncated simplex category.